src/summary/LongitudinalTableVisualizer.jsx

```diff
@@ -66,13 +66,13 @@ export default class LongitudinalTableVisualizer extends Visualizer {
         const finalArray = _.concat(favsArray,notFavsArray); //but the favs back together with the not favs in one array
         this.setState({ data: finalArray });
     }
-    componentWillReceiveProps(nextProps) {
+    componentWillReceiveProps = (nextProps) => {
```

Comment:
[JA1]: I think since we don't want to change the React lifecycle methods, this shouldn't be an arrow function.

Reply:
you're right, I must've changed that one by accident.  Fixed it now, thanks!